process/check.py

```diff
@@ -419,3 +427,42 @@ def _process_service_check(self, name, nb_procs, bounds, tags):
             tags=service_check_tags,
             message=message_str % (status_str[status], nb_procs, name)
         )
+
+    @staticmethod
+    def add_children(parent_pids):
+        """
+        Searches children pids for each of parent
+        pids recursively.
+        :param parent_pids: set of parent pids
+        :return: set parent and children pids
+        """
+        pids = set()
+        for ppid in parent_pids:
+            try:
+                parent = psutil.Process(ppid)
+                children = parent.children(recursive=True)
```

Comment:
Could you add a Debug log line here to help support ?

something like:
self.log.debug("found %s children for parent process %s", len(children), ppid)